config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -677,6 +681,14 @@ public EventProcessingConfigurer usingSubscribingEventProcessors() {
         return this;
     }
 
+    @Override
+    public EventProcessingConfigurer usingSubscribingEventProcessors(
+            SubscribableMessageSourceDefinitionBuilder builder) {
```

Comment:
Super nit: I would rename the parameter the same as in the interface.
```suggestion
            SubscribableMessageSourceDefinitionBuilder defaultSource) {
```